Source/UI/Miscellaneous/IPFHelpViewController.m

```diff
@@ -1,5 +1,4 @@
 #import "IPFHelpViewController.h"
-
```

Comment:
The reason I add a new line here is to distinguish between your own header and the headers you import from elsewhere